README.md

```diff
@@ -1,11 +1,17 @@
 # Text Classification Engine
 
 Look at the following
-[tutorial](http://predictionio.incubator.apache.org/demo/textclassification/)
+[tutorial](https://predictionio.apache.org/demo/textclassification/)
 for a Quick Start guide and implementation details.
 
 # Release Information
 
+### v0.13.0
+
+- Update for Apache PredictionIO 0.13.0
+- Fix warnings and use of case class
+- Python3 support in helper scripts
```

Comment:
Is that correct?

Reply:
![2018-09-20 20 01 49](https://user-images.githubusercontent.com/1094760/45814266-1557c480-bd10-11e8-857f-1c71c1fdd868.png)

It seems to be committed after v6.0 release.

Reply:
I understand the desire to make it consistent with other official templates, but going backward in version number feels strange to me.

Reply:
I couldn't agree more. 😕

Reply:
I think this is a time to start new history.